molecule/resources/converge.yml

```diff
@@ -6,7 +6,7 @@
 
   pre_tasks:
```

Comment:
Unfortunately I left a mistake on line 18:

```
failed_when: systemctl_status.rc > 1
```

Should be:
```
failed_when: systemctl_status.rc > 0
```

Reply:
thanks for spotting that! I'll fix now